knplabs/knp-snappy-bundle/1.4/etc/packages/knp_snappy.yaml

```diff
@@ -0,0 +1,10 @@
+# Read Documentation: https://github.com/KnpLabs/KnpSnappyBundle/blob/v1.4/README.markdown
+knp_snappy: ~
+#    temporary_folder: %kernel.cache_dir%/snappy
+#    pdf:
+#        enabled:    true
+#        options: []
+#    image:
+#        enabled:    true
+#        options: []
```

Comment:
@stof i don't understand what i should provide as conf settings, all settings in this bundle have a default value. 
Should i provide an empty conf like @20uf propose or the most used settings like @fabpot propose in another PR https://github.com/symfony/recipes-contrib/pull/53